chapters/pipelines.txt

```diff
@@ -3477,7 +3477,7 @@ OpDecorate %2 SpecId 12 ; decorate our 32-bit floating-point constant
 ---------------------------------------------------
 
 From the above we have two specialization constants, one is a signed 32-bit
-integer and the second is a 32-bit floating-point.
+integer and the second is a 32-bit floating-point number.
```

Comment:
We typically say "floating-point value".

Reply:
Fixed!